Sources/Orbit/Support/Images/Images.xcassets/elevation.imageset/Contents.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
If this proves useful, we can later create a folder in `Images/SnapshotReferences/` to not confuse it with a "real" image.

Reply:
Hmm isn't there a designated folder for "preview content"? I remember it being in Xcode projects at least... I'd hope we. can exclude that from distribution..

Reply:
Project have it and it is very useful, but packages do not still have this feature.

But we can create this folder `Preview Content` manually and mark it as excluded in the package manifest.

Reply:
But then we wouldn't be able to access the contents...?

Reply:
If thats the case, maybe we can even remove it (it served its purpose) and let previews/snapshots only show our version of both pre-rendered and realtime shadows (they are currently removed).

If we would use these design snapshots more, it would just make the package grow in size even more.

Reply:
Removed, both prerendered and live previews are now shown.